buildSrc/src/main/groovy/Rspec.groovy

```diff
@@ -52,6 +60,14 @@ class Rspec extends DefaultTask {
                 "-I", "${project.getProjectDir()}/client/ruby"
         ]
 
+        if (fileOutput) {
+            rspec_args.add("--format")
+            rspec_args.add("h")
+            rspec_args.add("--force-color")
+            rspec_args.add("--out")
+            rspec_args.add("rspec_output.html")
```

Comment:
Should probably drop this in build/reports/tests/rspec/index.html following along with what junit (attempts?) to do.

Reply:
Done